### ISSUE TITLE
Data breach at ADAC

### ISSUE THREAD

**Issue by swidup** [vz-risk/VCDB#6579]
https://translate.google.com/translate?sl=auto&tl=en&js=y&prev=_t&hl=en&ie=UTF-8&u=http%3A%2F%2Fwww.autobild.de%2Fartikel%2Fadac-mitgliederdaten-im-internet-7106293.html&edit-text=

**Comment by swidup** [comment]
Original link:  http://www.autobild.de/artikel/adac-mitgliederdaten-im-internet-7106293.html  The link above is the google translated link.

**Comment by swidup** [comment]
SERVICE-TOOLS
Insurance Rates
Article download
Used car market
Summer tires
New car configurator
All Topics
NEWS: LATEST ARTICLE
Audi A4: Euro NCAP crash test - A4 shines in a crash
VW Caddy: Euro NCAP crash test - Four stars for the Caddy
Porsche Macan (Update 2015) - Update for the Macan
Mazda CX-9: LA Auto Show 2015 - New, large Mazda SUV
Euro NCAP crash test: Honda, Audi, VW - A Small very big
News: All posts at a glance
ADAC: Membership Data on the Internet - 03/11/2015
Data breach at ADAC
Apparently weeks data from ADAC members were available on the Internet. The corresponding page is now the network.
(dpa / mas) The ADAC has data members appear for weeks openly readable on the web. A spokesman for Europe's largest-Car Club has confirmed this. Who since the beginning of October 2015 the on-line application for an ADAC credit card any membership number was entered, the saw also the associated name, the entry date and the type of membership, the spokesman said. Although addresses, bank details or other sensitive information were not displayed, it is possible with such data to drive abuse. The Automobile Club took the banner for the request of the credit card in the meantime from the mains. IMAGE, SWR and the "Allgemeine Zeitung" Mainz were first on the glitch reported.
Internet Expert: Form "gross negligence" 
Discovered she had the Mainzer Internet expert Tobias Huch. "I wanted to order the ADAC credit card. Then I saw that unencrypted data is transmitted," said the board member of the Young Liberals Rheinland-Pfalz in Mainz. "I consider this form of gross negligence." Huch is owner of a media agency.
All news and information to the ADAC 

ADAC spokesman Alexander Machowetz said in Munich, the Automobile Club have the application site immediately taken offline, contact Huch and check now how many members were affected. Everyone should soon be informed personally in order to prevent possible abuse. The data gap was due to process errors.
